src/qat/compiler/validation_passes.py

```diff
@@ -128,13 +129,23 @@ def run(self, builder: InstructionBuilder, res_mgr: ResultManager, *args, **kwar
             # Check if we've got a measure in the middle of the circuit somewhere.
             elif isinstance(inst, Acquire):
                 for qbit in model.qubits:
-                    if qbit.get_measure_channel() == inst.channel:
```

Comment:
Is there any reason this was previously implemented as `get_measure_channel` instead of `get_acquire_channel`?

Reply:
I'm not too sure if this was a mistake or intentional but I'll try to find out. Thinking the check might need to be more robust (e.g. look for acquires measures)